<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeliveryForm</class>
 <widget class="QWidget" name="DeliveryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询快递信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
                QLabel {
                    color: #333;
                    font: 14px &quot;幼圆&quot;;
                    font-weight: bold;
                }

                QLineEdit,
                QComboBox  {
                    padding: 8px;
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QTableWidget {
                    font: 14px &quot;幼圆&quot;;
                    background-color: #cdefff;
                }

                QTableWidget::item {
                    font: 14px &quot;幼圆&quot;;
                }

                QHeaderView {
                    font: bold 16px &quot;幼圆&quot;;
                }
                
                QTableWidget::item:selected {
                    background-color: #a6a6a6;
                }

                QTableWidget QHeaderView::section {
                    background-color: #65a3ff;
                    color: white;
                    font-weight: bold;
                    font-size: 16px;
                    padding: 4px;
                }

                QTableWidget::item {
                    padding: 10px;
                }

                QPushButton {
                    padding: 8px 16px;
                    font-size: 15px;
                    font-weight: bold;
                    border: 1px solid #4CAF50;
                    border-radius: 4px;
                    color: #fff;
                    background-color: #4CAF50;
                }

                 QPushButton:hover {
                    background-color: #45a049;
                    border: 1px solid #45a049;
                }

                QTextEdit {
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QScrollBar:vertical {
                    border: 1px solid #ccc;
                    background: #f0f0f0;
                    width: 10px;
                    margin: 0px 0px 0px 0px;
                }

                QScrollBar::handle:vertical {
                    background: #4CAF50;
                    min-height: 20px;
                }
            </string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>快递信息</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>721</width>
      <height>421</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>快递单号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>完整路线</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>省份</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>地级市</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目前状态</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>601</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="parcelIdLabel">
       <property name="text">
        <string>快递单号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="parcelIdInput">
       <property name="placeholderText">
        <string>在此输入有效快递单号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="queryBtn">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>111</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>返回上一级</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
